server/@types/models/Referral.ts

```diff
@@ -6,6 +6,21 @@ import type { TagColour } from '@accredited-programmes/ui'
 import type { User } from '@manage-users-api'
 import type { Prisoner } from '@prisoner-search'
 
+const referralStatuses = [
```

Comment:
This list of available statuses does need to be kept up to date and we will also need to change which statuses get included in the `Open` and `Closed` tabs on the My referral page.  See https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/blob/main/server/controllers/refer/caseListController.ts#L26-L28